gaming/schema.bash

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+#
+# Copyright 2022 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# A convenience script to help developers easily set up emulator, finapp server
+# and workload.
+
+wrench migrate up --directory ./schema
```

Comment:
How does this migration work? Will it apply the migrations in the files below as one DDL batch? Or will it execute each statement individually? Whenever possible, we should try to make our samples execute DDL statements in one batch, as that is significantly quicker than separate statements.

Reply:
AFAIK, wrench applies each file in a separate batch. I used this method to eventually highlight the use of such migration tools.

I've also provided two separate schema files (players.sql and trading.sql) for use in codelabs that leverage the gcloud command line instead of wrench.

Reply:
Makes sense.